warble/functions.cc

```diff
@@ -24,9 +24,35 @@ std::vector<std::string> FindHashtag(Database* db, std::string hashtag) {
         warbles.push_back(id);
       }
   }
+  LOG(INFO) << "Found warble ids : " << warbles.size() << " (" << hashtag << ")";
   return warbles;
 }
 
+bool Stream(Database* db, Any req, Any* rep) {
+  StreamRequest request;
+  StreamReply reply;
+  req.UnpackTo(&request);
+  std::string hashtag = request.hashtag();
+  std::vector<std::string> ids = FindHashtag(db,hashtag);
+  
+  if (ids.empty()) {
+    LOG(INFO) << "No warbles found for hashtag " << hashtag;
```

Comment:
This log may not be appropriate as streaming is not concerned with the initial warbles found for a hashtag.